### PR TITLE
fetch organisation member user info

### PR DESCRIPTION
**Pull request by danvixent**

### CODE REVIEW

File: datastore/mongo/organisation_member.go [hunk 1]
```diff
@@ -194,8 +200,93 @@ func (o *orgMemberRepo) FetchOrganisationMemberByUserID(ctx context.Context, use
...
+		return nil, datastore.ErrOrgMemberNotFound
 	}
 
+	err = o.fillOrgMemberUserMetadata(ctx, []*datastore.OrganisationMember{member})
 	return member, err
 }
+
+func (o *orgMemberRepo) fillOrgMemberUserMetadata(ctx context.Context, members []*datastore.OrganisationMember) error {
+	userIDs := make([]string, 0, len(members))
+	for i := range members {
```
Comment (Subomi) [inline datastore/mongo/organisation_member.go:212]: @danvixent Why can't this `for loop` be one?

**Review by Subomi — CHANGES_REQUESTED** [review]
@danvixent FYI

File: datastore/mongo/organisation_member.go (hunk 1 above)
Comment (danvixent) [reply datastore/mongo/organisation_member.go]: fixed

File: datastore/mongo/organisation_member_test.go [hunk 2]
```diff
@@ -200,17 +275,35 @@ func TestFetchOrganisationMemberByID(t *testing.T) {
...
+		LastName:  user.LastName,
+		Email:     user.Email,
+	}, *member.UserMetadata)
 }
 
 func TestFetchOrganisationMemberByUserID(t *testing.T) {
-	db, closeFn := getDB(t)
-	defer closeFn()
+	db, _ := getDB(t)
+	//defer closeFn()
```
Comment (jirevwe) [inline datastore/mongo/organisation_member_test.go:288]: Why is this line commented out? You're also ignoring the second value from the `getDB()` function

**Review by jirevwe — CHANGES_REQUESTED** [review]

File: datastore/mongo/organisation_member_test.go (hunk 2 above)
Comment (danvixent) [reply datastore/mongo/organisation_member_test.go]: Missed that, i used it for testing. Fixed now.

**Review by jirevwe — APPROVED** [review]

**Review by Dotunj — APPROVED** [review]

**Pull request merged**